### PR TITLE
EPMRPP-78274 || Upgrade react version to 17th

### PR DESCRIPTION
**Pull request by Vadim73i**

### CODE REVIEW

**Review by chivekrodis — COMMENTED** [review]
PR contains breaking changes
pipeline failed
tooltips(popper-js) are have wrong behaviour doesn't work or appear in the wrong place(f.e login page versions, select project, history)
upload btn broken
Looks like dashboards work inappropriate way(appearance, creating)
a lot of errors in the console

**Review by chivekrodis — CHANGES_REQUESTED** [review]
Please resolve conflicts
Have no possibility create Launch statistics chart
Edit Launch statistics chart leads to crash
Charts have strange behavior: widgets collapse in width after 1083px
Filter by start time on the launch page leads to crash app

**Review by chivekrodis — APPROVED** [review]

File: app/src/pages/admin/allUsersPage/allUsersGrid/projectsAndRolesColumn/projectsAndRolesColumn.jsx [hunk 1]
```diff
@@ -206,6 +207,7 @@ export class ProjectsAndRolesColumn extends Component {
       id: 'confirmationModal',
       data: {
         message: intl.formatMessage(messages.unassignModalText, {
+          b: (data) => DOMPurify.sanitize(`<b>${data}</b>`),
```
Comment (AmsterGet) [inline app/src/pages/admin/allUsersPage/allUsersGrid/projectsAndRolesColumn/projectsAndRolesColumn.jsx:210]: Could you please explain this fix?
It is not clear.

**Comment by AmsterGet** [comment]
Also please resolve conflicts.

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**